### PR TITLE
Create pip-21.md [unbond token can rebond at any time.]

### PR DESCRIPTION
**Pull request by laosiji-io**
unbond token can rebond at any time.

### CODE REVIEW

File: PIPs/pip-21.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+pip: 21
+title: unbond token can rebond at any time. 
+author: hui (@laosiji-io)
+status: Draft
+type: Standards
+category: validator
```
Comment (Ja7ad) [inline PIPs/pip-21.md:7]: change category to `core`

File: PIPs/pip-21.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+created: 15-01-2024
+---
+
+## Abstract
+
+The purpose of this proposal is to make the staking token more liquid.
+
+## Motivation
+
+Because the time of unbond_interval takes too long, and when the token is unboned, these token will not have any effect.
```
Comment (Ja7ad) [inline PIPs/pip-21.md:17]: Please add more details for motivation and specification

Comment (laosiji-io) [reply PIPs/pip-21.md]: added what I have temporarily thought of

**Comment by Ja7ad** [comment]
@laosiji-io when complete this proposal for publish draft?

**Comment by laosiji-io** [comment]
Do you mean you still need to complete the code writing?
Sorry, this is just one of my thoughts,I think this proposal is quite beneficial.
If feasible, let your professional team handle the code writing.

**Review by Ja7ad — APPROVED** [review]

**Pull request merged**